### PR TITLE
0.3.0

### PR DESCRIPTION
**Pull request by ben-gibson**

### CODE REVIEW

**Comment by ben-gibson** [comment]
It might be good to integrate this repository with a build tool like [travis](https://travis-ci.com) that will include the test results within the PRs etc.

File: tests/Sourcing/EventSteamTest.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+namespace Krixon\DomainEvent\Test\Sourcing;
+
+use InvalidArgumentException;
+use Krixon\DomainEvent\Event;
+use Krixon\DomainEvent\Sourcing\EventStream;
+use Krixon\DomainEvent\Sourcing\EventStreamId;
+use PHPUnit\Framework\TestCase;
+
+class EventSteamTest extends TestCase
```
Comment (krixon) [inline tests/Sourcing/EventSteamTest.php]: Typo in the class name.

```suggestion
class EventStreamTest extends TestCase
```

File: tests/Sourcing/EventSteamTest.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+{
+    public function testRejectsSteamWithNoEvents() : void
```
Comment (krixon) [inline tests/Sourcing/EventSteamTest.php]:
```suggestion
    public function testRejectsStreamWithNoEvents() : void
```

File: tests/Sourcing/EventSteamTest.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Event stream must contain at least one event.');
+
+        $steamId = new EventStreamId('foo');
```
Comment (krixon) [inline tests/Sourcing/EventSteamTest.php]:
```suggestion
        $streamId = new EventStreamId('foo');
```

File: tests/Sourcing/EventSteamTest.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        new EventStream($steamId, []);
+    }
+
+
+    public function testRejectsEventNumberLessThanZero() : void
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('First event number cannot be less than 0.');
+
+        $steamId = new EventStreamId('foo');
```
Comment (krixon) [inline tests/Sourcing/EventSteamTest.php]:
```suggestion
        $streamId = new EventStreamId('foo');
```

**Review by krixon — CHANGES_REQUESTED** [review]

File: tests/Sourcing/EventSteamTest.php (hunk 1 above)
Comment (ben-gibson) [reply tests/Sourcing/EventSteamTest.php]: Fixed

File: tests/Sourcing/EventSteamTest.php (hunk 2 above)
Comment (ben-gibson) [reply tests/Sourcing/EventSteamTest.php]: Fixed

**Review by krixon — DISMISSED** [review]

**Review by krixon — APPROVED** [review]

**Pull request merged**